ote/src/clj/ote/authorization.clj

```diff
@@ -6,6 +6,10 @@
             [taoensso.timbre :as log]
             [ote.nap.users :as users]))
 
+(defn require-transit-authority [user]
```

Comment:
So there's two options to handle a HTTP request fail: throwing and exception or returning a response object?
I suppose both are ok and it's a matter of design which is used in the app.

Admin refactoring uses the latter approach: a function is called to resolve an authentication fail response map, if it's not returned then a response is requested from a domain function.
Should same design be used here? 
services.admin uses `authorization-fail-response` to return `(http/transit-response "Not authorized. Bad role." 403)` which could be used here as well.
Additional upside there is that request status code is explicitly defiened, with `SecurityException` you have to find out what it will be.

Reply:
My bad. Will change.